itest/riak_kv_sweeper_SUITE.erl

```diff
@@ -433,19 +438,13 @@ scheduler_remove_participant_test(Config) ->
     riak_kv_sweeper:add_sweep_participant(SP1),
     riak_kv_sweeper:enable_sweep_scheduling(),
 
+    tick_per_index(Indices),
     receive {From, WaitIndex} ->
             riak_kv_sweeper:remove_sweep_participant(sweep_observer_1),
             From ! {TestCasePid, continue}
     end,
 
-    ok = receive_msg({ok, failed_sweep, sweep_observer_1, WaitIndex}, min_scheduler_response_time_msecs()),
-    #sweep{results = Result} = get_sweep_on_index(WaitIndex),
-    {ok,{_, fail}} = dict:find(sweep_observer_1, Result),
```

Comment:
@nickelization - Think we shouldn't remove the `{ok, {_, fail}}` match. The intentions is to capture the return and document the return value. It can actually be made stricter:  `{ok,{{_, _, _}, fail}}`.

The matching makes this test case unstable, but I think that can be addressed with doing a `match_retry(...)` instead.